### DISCUSSION TITLE
Getting the last version of msp430

### DISCUSSION THREAD

**Discussion by tunguskar** [compiler-explorer/compiler-explorer#6753] [Q&A]
Hi,

The  last version of MSP430 GCC compiler is 9.3 https://www.ti.com/tool/MSP430-GCC-OPENSOURCE

In the list on Compiler Explorer the last one is 6.2.1

Would it be possible to integrate the latest version?

Kind regards

Christian

**Comment by mattgodbolt** [comment]
Hi! Sorry we didn't see this! https://github.com/compiler-explorer/compiler-explorer/issues/new/choose is a good place to ask these kinds of questions (picking "Compiler request"). However, I've filed one as: #7076
